builtin/rebase.c

```diff
@@ -26,6 +27,31 @@ static int use_builtin_rebase(void)
 	strbuf_release(&out);
 	return ret;
 }
+static int finish_rebase(){
+
+}
+
+static int run_specific_rebase(const char *command_line)
+{
+	struct argv_array child_env = ARGV_ARRAY_INIT;
+	const char *child_argv[] = { NULL,NULL };
+	int status;
+
+	fprintf(stderr, "Executing %s\n", command_line);
+	child_argv[0] = command_line;
+	argv_array_pushf(&child_env, "--preserve-merges", NULL);
+	status = run_command_v_opt_cd_env(child_argv, RUN_USING_SHELL, NULL,
+					 child_env.argv);
+	if(status == 0) {
+		finish_rebase();
+	}
+	else if(status == 2) {
+		apply_autostash();
+		remove_dir_recursively("GIT_DIR=%s/rebase-apply" ||
+					"GIT_DIR=%s/rebase-merge");
```

Comment:
remove_dir_recursively() takes 2 arguments.

I don't see how this can even compile.